torchmdnet/datasets/ace.py

```diff
@@ -47,7 +47,7 @@ class Ace(MemmappedDataset):
     - `positions`: Atomic positions. Units: Angstrom.
     - `forces`: Forces on the atoms. Units: eV/Å.
     - `partial_charges`: Atomic partial charges. Units: electron charges.
-    - `dipole_moment` (version 1.0) or `dipole_moments` (version 2.0): Dipole moment (a vector of three components). Units: e*Å.
+    - `dipole_moment` (version 1.0) or `dipole_moment` (version 2.0): Dipole moment (a vector of three components). Units: e*Å.
```

Comment:
Are you sure the "s" should not be there?
Not it just says dipole_moment twice. I probably made that distinction in the docstring for a reason.

Reply:
Yes you're right. The layout version 2.0 of ace it's using `dipole_moments`. I considered that in the code but I changed the name in the documentation.